src/jobs/run_tests.yml

```
@@ -0,0 +1,30 @@
+description: Start Android instance, test your app and stop instance.
+
+executor: default
+
+parameters:
+  recipe_uuid:
+    type: string
+    default: ""
+    description: |
```

```suggestion
    description: >-
```
To make it appear as required on the online documentation.